src/components/backend-ai-registry-list.ts

```diff
@@ -609,7 +695,14 @@ class BackendAIRegistryList extends BackendAIPage {
     );
   }
 
```

Comment:
Please refer [here(`render()` declared in Lit-Element)](https://github.com/lit/lit/blob/1ac58058fbdf58633b2bb8d10649d0e11c409256/packages/lit-element/src/lit-element.ts#L217-L227).